### ISSUE TITLE
is there tsop48 convertor schematic?

### ISSUE THREAD

**Issue by youxiaojie** [radiomanV/TL866#54]
is there tsop48 convertor schematic?

**Comment by radiomanV** [comment]
Sorry for late comment, you mean this: https://proghq.org/wiki/TL866_TSOP48_adapter ?

**Comment by youxiaojie** [comment]
does this 48pin adapter suitable to your software?

**Comment by radiomanV** [comment]
Yes, the ATtiny chip on the adapter can be loaded with the firmware here: https://github.com/radiomanV/TL866/tree/master/TSOP_Encryption   .
Note taht this adapter will work only with TL866A/CS and TL866II+ programmers.
